File: backend/src/main/java/isep/webapp/terasync/service/odSystem/PackagingService.java
package isep.webapp.terasync.service.odSystem;

import isep.webapp.terasync.model.odSystem.Packaging;
import isep.webapp.terasync.repository.odSystem.PackagingRepository;
import isep.webapp.terasync.service.EntityService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;

@Service
public class PackagingService extends EntityService<Packaging, PackagingRepository> {
    @Autowired
    public PackagingService(PackagingRepository entityRepository) {
        super(entityRepository);
    }

    @Transactional
    @Override
    public Integer deleteEntityById(Integer id) {
        return entityRepository.deleteEntityById(id);
    }
}

File: front/src/app/service/odSystem/packaging.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { Packaging } from '../../model/odSystem/packaging';
import { EntityService } from '../entity.service';

@Injectable({
  providedIn: 'root'
})
export class PackagingService extends EntityService<Packaging> {

  constructor(http: HttpClient) {
    super(http, "packaging");
  }
}
